.kokoro/docker/docs/Dockerfile

```diff
@@ -64,4 +64,12 @@ RUN wget -O /tmp/get-pip.py 'https://bootstrap.pypa.io/get-pip.py' \
   && python3.8 /tmp/get-pip.py \
   && rm /tmp/get-pip.py
 
+# Install gcloud SDK
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | \
+    tee -a /etc/apt/sources.list.d/google-cloud-sdk.list \
+  && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | \
+    apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - \
+  && apt-get update -y \
+  && apt-get install python2 google-cloud-sdk -y
```

Comment:
do we need `python2` if we don't need to run system tests?

Reply:
This PR isn't trying to fix #832 (at least not yet).  To do that, we need either to have a `.kokoro/build-systests.sh` script, which runs only the systests, or else work out some environment variable which we use to turn them on / off in `.kokoro/build.sh`.

Either way will require adding something like `.kokoro/presubmit/system-3.8.cfg`, to either set the environment variable or trigger the alternate script, which will then need a corresponding tweak to the `google3` Kokoro configuration for this repostitory.

Reply:
Currently it just skips the system tests (missing 3.7 interpreter), and `docs` session runs fine. I think it's fine to move forward as-is.

Reply:
PR #833 is based on this PR:  once we merge here, I will rebase it against `master`, and then wait to merge until the `google3` changes are in place to enable the `Kokoro system-3.7` build.